.freebsd.test.py

```diff
@@ -31,13 +32,16 @@
 
 def main():
     judgeenv.env['runtime'] = {}
-    judgeenv.env['extra_fs'] = {'PERL': ['/dev/dtrace/helper$'], 'RUBY2': ['/dev/dtrace/helper$']}
+    judgeenv.env['extra_fs'] = {
+        'PERL': [ExactFile("/dev/dtrace/helper")],
+        'RUBY2': [ExactFile("/dev/dtrace/helper")]
```

Comment:
Maybe the YAML format could be something like:

```yaml
extra_fs:
  PERL:
  - file: /dev/dtrace/helper
  - dir: /some/exact/directory
  - recurse: /some/directory/and/all/children
```

Reply:
sounds good to me, should I do that?

Reply:
Agree, though I'd prefer keeping the same names as the code (ExactDir, etc.)

Reply:
+1 on that. we could change the naming scheme to `exact_dir` and `recursive_dir`

Reply:
OK, use `exact_dir` and `recursive_dir` then.